src/app/pages/apps/store/docker.store.ts

```diff
@@ -79,9 +79,17 @@ export class DockerStore extends ComponentStore<DockerConfigState> {
     return this.api.call('docker.status');
   }
 
-  setDockerPool(poolName: string | null): Observable<Job<DockerConfig>> {
+  setDockerPool(poolName: string | null, migrateApps?: boolean): Observable<Job<DockerConfig>> {
+    const payload = {
+      pool: poolName,
+    } as DockerConfig;
+
+    if (typeof migrateApps === 'boolean') {
```

Comment:
Insignificant but you can just check for a truthy `migrateApps` instead of a type check. The only case where `payload.migrate_applications` should be true is when `migrateApps` is true. If it's `false` or `undefined`, `payload.migrate_applications` should be false anyway.